TestCases/turbomachinery/Aachen_turbine/aachen_3D_MP.cfg

```diff
@@ -0,0 +1,371 @@
+MARKER_HEATFLUX= (BLADE1, 0.0, BLADE2, 0.0, BLADE3, 0.0, HUB1, 0.0, SHROUD1, 0.0, HUB2, 0.0, SHROUD2, 0.0, HUB3, 0.0, SHROUD3, 0.0)
+%
+% Periodic boundary marker(s) (NONE = no marker)
+% Format: ( periodic marker, donor marker, rot_cen_x, rot_cen_y, rot_cen_z, rot_angle_x-axis, rot_angle_y-axis, rot_angle_z-axis, translation_x, translation_y, translation_z)
+MARKER_PERIODIC= (PER1_STATOR1, PER2_STATOR1, 0.0, 0.0, 0.0, 0.0, 0.0, 8.7804878, 0.0, 0.0, 0.0, PER1_ROTOR, PER2_ROTOR, 0.0, 0.0, 0.0, 0.0, 0.0, 8.7804878, 0.0, 0.0, 0.0, PER1_STATOR2, PER2_STATOR2, 0.0, 0.0, 0.0, 0.0, 0.0, 8.7804878, 0.0, 0.0, 0.0)
+%
+%
+%-------- INFLOW/OUTFLOW BOUNDARY CONDITION SPECIFIC FOR TURBOMACHINERY --------%
+%
+% Inflow and Outflow markers must be specified, for each blade (zone), following the natural groth of the machine (i.e, from the first blade to the last)
+MARKER_TURBOMACHINERY= (INFLOW_STATOR1, OUTFLOW_STATOR1, INFLOW_ROTOR, OUTFLOW_ROTOR, INFLOW_STATOR2, OUTFLOW_STATOR2)
+MARKER_ANALYZE = (INFLOW_STATOR1, OUTFLOW_STATOR2)
+% Mixing-plane interface markers must be specified to activate the transfer of information between zones
+MARKER_MIXINGPLANE_INTERFACE= (OUTFLOW_STATOR1, INFLOW_ROTOR, OUTFLOW_ROTOR, INFLOW_STATOR2)
+% Mixing-plane interface markers must be specified to activate the transfer of information between zones
+MARKER_ZONE_INTERFACE= (OUTFLOW_STATOR1, INFLOW_ROTOR, OUTFLOW_ROTOR, INFLOW_STATOR2) 
+%
+% Non reflecting boundary condition for inflow, outfolw and mixing-plane
+% Format inlet:  ( marker, TOTAL_CONDITIONS_PT, Total Pressure , Total Temperature, Flow dir-norm, Flow dir-tang, Flow dir-span, under-relax-avg, under-relax-fourier)
+% Format outlet: ( marker, STATIC_PRESSURE, Static Pressure value, -, -, -, -, under-relax-avg, under-relax-fourier)
+% Format mixing-plane in and out: ( marker, MIXING_IN or MIXING_OUT, -, -, -, -, -, -, under-relax-avg, under-relax-fourier)
+MARKER_GILES= (INFLOW_STATOR1, TOTAL_CONDITIONS_PT, 158245.38, 308.26, 1.0, 0.0, 0.0, 0.3, 0.0, OUTFLOW_STATOR1, MIXING_OUT, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3, 0.0, INFLOW_ROTOR, MIXING_IN, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3, 0.0, OUTFLOW_ROTOR, MIXING_OUT, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3, 0.0, INFLOW_STATOR2, MIXING_IN, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3, 0.0, OUTFLOW_STATOR2, STATIC_PRESSURE_1D, 110050.96, 0.0, 0.0, 0.0, 0.0 , 1.0, 0.0)
+SPATIAL_FOURIER= NO
+%
+% This option insert an extra under relaxation factor for the Giles BC at the hub and shroud levels
+GILES_EXTRA_RELAXFACTOR= (0.05, 0.05)
+%
+%---------------------------- TURBOMACHINERY SIMULATION -----------------------------%
+%
+% Format: (marker)
+% If the ROTATING_FRAME option is activated, this option force
+% the velocity on the boundaries specified to 0.0
+MARKER_SHROUD= (SHROUD1, SHROUD2, SHROUD3)
+%
+% Specify kind of architecture (AXIAL, CENTRIPETAL, CENTRIFUGAL, CENTRIPETAL_AXIAL)
+TURBOMACHINERY_KIND= AXIAL AXIAL AXIAL 
+%
+% Uncomment to work with new_turbo_outputs
+%TURBO_PERF_KIND= (TURBINE, TURBINE, TURBINE)
```

Comment:
I think updating the comment here would be better to avoid confusiing when this is merged into develop

Reply:
Hey Josh, what would you recommend instead?

Reply:
I went with this #2011